CROWDIN.md

```diff
@@ -0,0 +1,62 @@
+# Crowdin Translator How-To
+
+Stellar Developer Docs use [Crowdin](https://crowdin.com/) for localization. This document provides steps and resources for managing Stellar Docusaurus Crowdin translations as a Crowdin user with a Translator role.
+
+---
+
+## Overview
+
+As new English source files are added to the Stellar Docusaurus Documentation GitHub repository, they are automatically uploaded to Crowdin. From there, an automated AI translation is run on the files to translate them into other target languages (currently only Spanish at the time of writing).
+
+In cases where the AI translation is inaccurate, a user with the Translator role will need to update and edit the translations manually.
+
+---
+
+## Translator User
+
+To get started, you need to set up a Crowdin account and be added to the Stellar Documents Crowdin project. This involves creating a Crowdin user account and requesting access to the project from a project owner. 
+
+For more details, see the following documentation: 
+[Create an Account](https://support.crowdin.com/for-translators/#create-an-account)
+
+---
+
+## Editor
+
+As a translator, you will have access to the editor, where you can make changes to the content. 
+
+**Note:** Crowdin uses the term "strings" for content being translated.
+
+You can access all translated documents via the Crowdin dashboard. From there, you can view each document in the editor. 
+
+For more details on working within the editor, see the following documentation: 
+[Working in the Editor](https://support.crowdin.com/for-translators/#working-in-the-editor) 
+
+For a more in-depth look, see: 
+[Online Editor Guide](https://support.crowdin.com/online-editor/)
+
+---
+
+## Glossary
+
+For common words or phrases, it is good practice to add them to the glossary. The glossary helps the AI translator provide more accurate translations. 
```

Comment:
looks like for our project, the glossary is "read-only" for any member who's not a manager or owner. i'm not exactly sure which of the glossary access settings are best for our use-case, though.

Reply:
@ElliotFriend I think it's possible to update Glossary permission: https://support.crowdin.com/glossary/#:~:text=To%20allow%20translators%20and%20proofreaders,access%20level%20for%20project%20members.

I'm looking into it now

Reply:
Ok Glossary is updated to allow members to edit.